composer.json

```diff
@@ -13,9 +13,9 @@
     "require": {
         "php": ">=7.0",
         "dragonbe/vies": "^1.0",
-        "symfony/http-kernel": "^2.8",
-        "symfony/symfony": "^2.8",
-        "symfony/validator": "^2.8"
+        "symfony/http-kernel": "^2.8|^3.0|^4.0",
+        "symfony/dependency-injection": "^2.8|^3.0|^4.0",
```

Comment:
:+1: goeie om dit strakker te zetten heel het framework is idd niet nodig

Reply:
Volgens mij kan de `http-kernel` dependency ook de deur uit? Ik zie 'm nergens actief gebruikt worden.

Reply:
hier wel : https://github.com/MyOnlineStore/ViesBundle/blob/master/src/DependencyInjection/SandwichViesExtension.php

Reply:
Dan ga ik dat meteen rechttrekken, je zou hier namelijk de `Symfony\Component\DependencyInjection\Extension\Extension` moeten extenden en (in dit geval) niet die van de HttpKernel. ;-)